### PR TITLE
PLANET-4348 Bring gutenberg plugin up to date with classic plugin

### PR DESCRIPTION
**Pull request by dpivo**
- Added un-migrated code
- Made sure styles are loading.

Ref: https://jira.greenpeace.org/browse/PLANET-4348

### CODE REVIEW

**Comment by comzeradd** [comment]
We could follow the same structure we have on blocks. So `campaign-en-block` is a mixin that is used for more than one campaign, so move it inside a `styles/campaigns/_mixins.scss` and include it from there.

File: react-blocks/src/styles/style.scss [hunk 1]
```diff
@@ -5,3 +5,13 @@
 @import "styleguide/src/base/mixins";
 
 @import "blocks/ENForm/components/enform";
+
+// Blocks
+@import "blocks/ENForm/components/enform";
```
Comment (comzeradd) [inline react-blocks/src/styles/style.scss]: Isn't that a duplicate of the line above?

**Comment by Inwerpsel** [comment]
The changes to the tease-thankyou.twig template look good to me.

File: classes/blocks/class-enform.php [hunk 2]
```diff
@@ -253,8 +302,10 @@ public function prepare_data( $attributes ): array {
 			[
 				'fields'       => $attributes,
 				'redirect_url' => isset( $attributes['thankyou_url'] ) ? filter_var( $attributes['thankyou_url'], FILTER_VALIDATE_URL ) : '',
-				'nonce_action' => 'enform_submit',
 				'form'         => $rendered_form,
+				'social'         => $social,
```
Comment (comzeradd) [inline classes/blocks/class-enform.php]: Can you align signs here?

**Comment by dpivo** [comment]
> Looks good 💯
> 
> Just a small indentation fix and it's good to go 🚀

@comzeradd Fixed 👌

**Pull request merged**